src/stores/ToteutussuunnitelmaStore.ts

```diff
@@ -9,20 +9,34 @@ Vue.use(VueCompositionApi);
 
 const logger = createLogger('Toteutussuunnitelma');
 
+interface ValidationCause {
```

Comment:
Pitäisiköhän tehdä amosaa-ui:n jokin oma types.ts tjms mihin kasattaisiini näitä tyypityksiä? Jos tähän esim tyypityksien määrä kasvaa niin lista on iso.

Reply:
Löytyi tyyppi frontendutilsista: https://github.com/Opetushallitus/eperusteet-frontend-utils/pull/286